Processors/Monolog/DeamonLoggerExtraWebProcessor.php

```diff
@@ -170,7 +170,7 @@ private function appendUserMethodInfo(UserInterface $user): void
      */
     private function isUserInstanceValid(?TokenInterface $token): bool
     {
-        return $token instanceof TokenInterface && $token->getUser() instanceof $this->userClass;
+        return $token instanceof TokenInterface && $this->userClass && $token->getUser() instanceof $this->userClass;
```

Comment:
can you add the null in the comparison ?
```suggestion
        return $token instanceof TokenInterface && null !== $this->userClass && $token->getUser() instanceof $this->userClass;
```